qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -78,12 +78,13 @@ These are the columns required for successfully submit your data to EBI:
 +----------------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
 | ``physical_specimen_location``   | free text               | Where you would go to find physical sample or DNA, regardless of whether it is still available or not.                                              |
 +----------------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``taxon_id``                     | integer                 | NCBI's taxon id for the sample. Note, for amplicon sequencing, this is the taxonomy id for the metagenome being targeted, not the host taxonomy id. |
-+----------------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``description``                  | free text               | Description of the sample.                                                                                                                          |
+| ``taxon_id``                     | integer                 | NCBI taxon ID for the sample. Should indicate metagenome being investigated. Examples: 410658 for soil metagenome, 749906 for gut metagenome,       |
+|                                  |                         | 256318 for metagenome (used for unspecified or blanks). To find others visit `The Taxonomy Database <http://www.ncbi.nlm.nih.gov/taxonomy>`__       |
 +----------------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
 | ``scientific_name``              | free text               | NCBI's scientific name for the provided taxon ID. Note, the name of the metagenome, not the host scientific name.                                   |
```

Comment:
Change

Note, the name of the metagenome, not the host scientific name.

to

Note: this is the name of the metagenome, not the host scientific name. Examples: soil metagenome, gut metagenome, marine sediment metagenome, marine metagenome.